src/System.Net.Primitives/tests/FunctionalTests/CookieCollectionTest.cs

```diff
@@ -152,5 +153,73 @@ public static void SyncRoot_Get_Success()
             CookieCollection cc = new CookieCollection();
             Assert.Equal(cc, cc.SyncRoot);
         }
+
+        [Fact]
+        public static void CopyTo_Array_Success()
+        {
+            CookieCollection cc = CreateCookieCollection1();
+            Array c = new object[cc.Count];
+            cc.CopyTo(c, 0);
+            EnsureEqual(cc, c);
+        }
+
+        [Fact]
+        public static void CopyTo_CookieArray_Success()
+        {
+            CookieCollection cc = CreateCookieCollection1();
+            Cookie[] c = new Cookie[cc.Count];
+            cc.CopyTo(c, 0);
```

Comment:
Note: `CopyTo(Cookie[])` doesn't exist in the [reference assembly contract](https://github.com/dotnet/corefx/blob/b84605422a74d69e7aa61a25d8760bfe2fa32296/src/System.Net.Primitives/ref/System.Net.Primitives.cs#L45-L56). If/when the implementation is updated to match the contract (see #2591), or the test project is updated to compile against the contract, this test will need to be removed.